### DISCUSSION TITLE
Disk space issue. Need to automate running commands after every deployment

### DISCUSSION THREAD

**Discussion by gyaaniguy** [dokku/dokku#7001] [Q&A]
These 2 commands are able to reclaim disk space

```bash
 dokku repo:gc PROJECT_NAME
 git -C /home/dokku/PROJECT_NAME repack -a -d -f --window=0
```

But currently I have to run these manually after every few deploys. 
Is there a way to automate these. So they are run after every deploy ? 

app.json postdeploy isn't applicable, this its context is only within the docker container and not the server host.

Thank you

**Comment by josegonzalez** [comment]
How much disk space does this clear? It seems strange that you'd have to run that to get back disk space - we don't touch the repo that much other than to receive your contents...
